cli/main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gluster/glusterd2/cli/cmd"
+)
+
+func main() {
+	// Migrate old format Args into new Format. Modifies os.Args()
```

Comment:
I moved initialization code before cmd initialization. Not yet made configurable. 

With the help of `PersistentPreRun` option in `cobra` I can initialize log there, but it is not possible to initialize log file before parsing the args. (Logger will not be available during Args Migrate). I need to write custom parser to parse `--log-file` and `--log-level` outside the `cobra.Command` initialization.

I will address this with separate patch, as of now added constants for the same.

I will address this in separate patch.